cirq-core/cirq/experiments/t1_decay_experiment.py

```diff
@@ -63,18 +63,21 @@ def t1_decay(
     """
     min_delay_dur = value.Duration(min_delay)
     max_delay_dur = value.Duration(max_delay)
+    min_delay_nanos = min_delay_dur.total_nanos()
+    max_delay_nanos = max_delay_dur.total_nanos()
 
     if repetitions <= 0:
         raise ValueError('repetitions <= 0')
+    if not isinstance(min_delay_nanos, float) or not isinstance(max_delay_nanos, float):
```

Comment:
think this needs to be `(float,int)` and not just `float`.  `isinstance(1, float)` returns `False`.

Reply:
Also we probably want to allow things like `np.int64`, `np.float64`, etc. It might be nice to define a tuple of allowed "numeric" types in one place and use it throughout cirq, though that could be a future PR.

Reply:
While, in general, this is true, I don't think it is true here, since we control cirq.Duration.total_nanos().  
I went ahead and modified cirq.Duration to guarantee returning a float, even when passed a numpy.int64 type.

Reply:
Actually, I take that back.  It looks like Duration is not typed correctly and can return an int.  I have made the typing more clear for Duration as well.